MAiD - Case Management App/force-app/main/default/reports/ICYMMHAReports.reportFolder-meta.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ReportFolder xmlns="http://soap.sforce.com/2006/04/metadata">
+    <folderShares>
+        <accessLevel>Manage</accessLevel>
+        <sharedTo>admin.user@gov.bc.ca.bcmohmaid</sharedTo>
+        <sharedToType>User</sharedToType>
+    </folderShares>
+    <folderShares>
+        <accessLevel>Manage</accessLevel>
+        <sharedTo>ICY_Business_Administrator</sharedTo>
```

Comment:
Please also add Manage shared to for ICY @lilianenassar-bib (one above ICY_Business_Administrator) @lilianenassar-bib